sonar-scm-svn-plugin/src/main/java/org/sonar/plugins/scm/svn/SvnPlugin.java

```diff
@@ -19,11 +19,14 @@
  */
 package org.sonar.plugins.scm.svn;
 
-import java.util.Arrays;
-import org.sonar.api.SonarPlugin;
-
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import org.sonar.api.SonarPlugin;
+import org.tmatesoft.svn.core.auth.ISVNAuthenticationManager;
+import org.tmatesoft.svn.core.wc.ISVNOptions;
+import org.tmatesoft.svn.core.wc.SVNClientManager;
+import org.tmatesoft.svn.core.wc.SVNWCUtil;
 
 public final class SvnPlugin extends SonarPlugin {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SonarPlugin"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)